modules/nw-kuryr-cleanup.adoc

```diff
@@ -0,0 +1,236 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/migrate-from-kuryr-sdn.adoc
+
+:_content-type: PROCEDURE
+[id="nw-kuryr-cleanup_{context}"]
+= Cleaning up resources after migration
+
+After migration from the Kuryr network plugin to the OVN-Kubernetes network plugin completes, clean up the resources that Kuryr created previously. 
+
+.Prerequisites
+
+* You installed the {product-title} CLI, `oc`.
+// TODO: replace jq
+* Install `jq` tool
+// TODO: Can we outsource some of the other prereqs to openstack CLI docs?
+* You installed a Python interpreter.
+* You installed the `openstacksdk` Python package.
```

Comment:
Openstacksdk cannot be treated as openstack client requirement, since as for now, there are limitations for openstack cli how it can be used (as for example filtering by tags) hence the python and openstacksdk.